specifyweb/frontend/js_src/lib/components/QueryBuilder/fromTree.ts

```diff
@@ -152,6 +152,9 @@ const defaultFields: RR<
         : []),
     ];
   },
+  TectonicUnit(nodeId, rankName) {
+    
+  }
```

Comment:
Not sure what goes here

Reply:
Nice catch here! 

Ah, this should return the fields for a CollectionObject query which when given `nodeId` and `rankName`. 
More specifically, it should return the COs which are associated to a particular TectonicUnit node. 

Though there's not a link from `CO -> TectonicUnit` right now, so this isn't really possible. 

Are there going to be any non-tree tables which have a relationship to TectonicUnit? 
If not, we should probably omit this tree from the `queryFromTree` types, and disable/remove the Query icon from the TectonicUnit TreeActions.

Reply:
I've added placeholder fields for now. Can be replaced later when we have the requirements